OmniKit/MessageTransport/MessageBlocks/StatusError.swift

```diff
@@ -9,16 +9,137 @@
 import Foundation
 
 public struct StatusError : MessageBlock {
+    // https://github.com/openaps/openomni/wiki/Command-02-Status-Error-response
     
-    public let blockType: MessageBlockType = .statusError
-    
-    public let data: Data
+//    public enum lengthType: UInt8{
+//        case normal = 0x10
+//        case configuredAlerts = 0x13
+//        case faultEvents = 0x16
+//        case dataLog = 0x04*numberOfWords+0x08
+//        case faultDataInitializationTime = 0x11
+//        case hardcodedValues  = 0x5
+//        case resetStatus = numberOfBytes & 0x03
+//        case dumpRecentFlashLog = 0x13
+//        case dumpOlderFlashlog = 0x14
+//
+    // public let numberOfWords: UInt8 = 60
+    // public let numberOfBytes: UInt8 = 10
+
+    public enum ProgressType: UInt8 {
+        case initialized = 0
+        case tankPowerActivated = 1
+        case tankFillCompleted = 2
+        case pairingSuccess = 3
+        case Purging = 4
+        case readyForInjection = 5
+        case injectionDone = 6
+        case primingCannula = 7
+        case runningNormal = 8
+        case runningLessThen50ULeftInReservoir = 9
+        case OnenotUsedButin33 = 10
+        case TwonotUsedButin33 = 11
+        case TheenotUsedButin33 = 12
+        case errorEventLoggedShuttingDown = 13
+        case alertExpiredDuringInitializationShuttingDown = 14
+        case podInactive = 15  // ($1C Deactivate Pod or packet header mismatch)
+    }
+
+    public enum DeliveryInProgressType: UInt8 {
```

Comment:
Looking on the wiki, it looks like this is a bitfield.  If that is the case, this should be an OptionSet